### ISSUE TITLE
Branching model needs improvement

### ISSUE THREAD

**Issue by r0h4n** [Tendrl/api#98]
Please follow below branching model for better categorization of code which will allow devs to push code in parallel without regressions:

http://nvie.com/posts/a-successful-git-branching-model/

Check how its done on the node-agent or the commons/ceph/gluster-integration repositories

**Comment by r0h4n** [comment]
Is this done?

**Comment by anivargi** [comment]
Develop was merged into master, we will follow the convention in the upcoming commits.
